oxidation/libparsec/crates/platform_device_loader/src/native/local_device_file.rs

```diff
@@ -0,0 +1,278 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
+
+use std::{
+    collections::HashSet,
+    ffi::OsStr,
+    fs::File,
+    path::{Path, PathBuf},
+};
+
+use libparsec_client_types::{
+    AvailableDevice, DeviceFile, DeviceFilePassword, DeviceFileRecovery, DeviceFileType,
+    LocalDevice, LocalDeviceError, LocalDeviceResult,
+};
+use libparsec_crypto::SecretKey;
+
+/// TODO: Remove me once wasm doesn't need to mock
+pub async fn test_gen_default_devices() {
+    panic!("This function shouldn't be called in native platform")
+}
+
+async fn _list_available_devices(config_dir: &Path) -> LocalDeviceResult<Vec<AvailableDevice>> {
```

Comment:
Could we not add the `_` prefix we have the `Vis` keyword for that

Reply:
I will rename `list_available_devices_core`